telemetry/definitions/commonDefinitions.json

```diff
@@ -52,6 +52,17 @@
             "type": "string",
             "description": "A generic version metadata"
         },
+        {
+            "name": "credentialModification",
+            "type": "string",
+            "allowedValues": ["Add", "Edit", "Delete"],
+            "description": "The type of modification performed on the credentials"
+        },
+        {
+            "name": "source",
```

Comment:
This will probably become a 'primitive' of telemetry at some point (e.g. not metadata)
Keeping track of an originator is extremely common in web browser telemetry as well as related protocols (think HTTP/HTTPS).
Just a little brain-dump.